models/customers-search-list-my-view/src/builder.spec.ts

```diff
@@ -0,0 +1,41 @@
+import CustomersSearchListMyView from './builder';
+import { TCustomersSearchListMyViewGraphql } from './types';
+
+describe('building', () => {
```

Comment:
It would be great if we can write more assertions in the tests.
We already found some inconsistencies in some of the models because their tests were too shallow so I think it's helpful if we try to populate all the model's properties and assert them.

[Here](https://github.com/commercetools/test-data/blob/main/models/product-projection/src/builder.spec.ts#L249) is an example of what I mean.

Reply:
Done https://github.com/commercetools/test-data/pull/654/commits/29a5a7e24f2bea40ab932efc3ada8db4b9edb70f. Let me know if this is enough

Reply:
Yes. Thanks! 👍